specs/src/specs/params.md

```diff
@@ -29,7 +29,7 @@ are blocked by the `x/paramfilter` module.
 | consensus.Version.AppVersion | 1 | Determines protocol rules used for a given height. Incremented by the application upon an upgrade. | False |
 | distribution.CommunityTax | 0.02 (2%) | Percentage of the inflation sent to the community pool. | True |
 | distribution.WithdrawAddrEnabled | true | Enables delegators to withdraw funds to a different address. | True |
-| distribution.BaseProposerReward | 0 | Reward in the mint demonination for proposing a block. | True |
+| distribution.BaseProposerReward | 0 | Reward in the mint denominations for proposing a block. | True |
```

Comment:
I think this can be singular

```suggestion
| distribution.BaseProposerReward | 0 | Reward in the mint denomination for proposing a block. | True |
```

Reply:
Thank you. I have corrected it.